app/src/main/java/com/tvtracker/ui/main/BrowseViewModel.kt

```diff
@@ -15,8 +15,6 @@ import kotlinx.coroutines.launch
 
 class BrowseViewModel(var mediaService: IMediaService = MediaService()): ViewModel() {
 
-    val PAGE_SIZE = 10
-
```

Comment:
This page size could be used as a constant, which is why I created a companion object and declared this as a constant inside it.

Reply:
Good idea...  but requires several more lines of code.  I could go either way on this.